src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/MethodologyStatusPage.tsx

```diff
@@ -40,6 +44,10 @@ const MethodologyStatusPage = () => {
 
   const [isEditing, toggleForm] = useToggle(false);
 
+  const { data: methodologyStatuses } = useQuery(
```

Comment:
I imagine you'll need to wrap any tests of this page in a query client provider now